### DISCUSSION TITLE
LOG SEVERITY DEBUG is not available?

### DISCUSSION THREAD

**Discussion by Marcus1911** [directus/directus#22018] [Feature Requests]
### Summary

I'm trying to setup a debug level severity but it seems is not implemented for .env. See figure:

![image](https://github.com/directus/directus/assets/5570044/554018eb-fd97-4051-98c2-0d2a6548aab4)

I've setup `LOG_LEVEL: DEBUG` in the .env 
 

### Basic Example

**Steps to reproduce:** 
 - Use docker-compose 
 - Create a environment variable as: `LOG_LEVEL: DEBUG`
 - Run the docker compose    

### Motivation

Essential for those who often work with Directus, particularly with its API, and wish to debug errors or comprehend complex behaviors.

### Detailed Design

Make all the log levels available is necessary for developers.

### Must Have:
- Log level DEBUG

### Should Have:
- All severities available

### Could Have:
- A rich documentation about log levels 

### Won't Have:
- Just documentation, without practical action in implementing

### Drawbacks

Is not commonly used by users or those who want just explore directus in a high level.

### Alternatives

The impact of not doing this is poor debugging and constantly difficulty in bind new feat.

### Adoption Strategy

A lot developers should enjoy this feature. It should be implemented using environment variables.

**Comment by br41nslug** [comment]
https://docs.directus.io/self-hosted/config-options.html#general

![image](https://github.com/directus/directus/assets/9389634/074ded02-ed18-423c-b58c-d64fe03deca0)

log levels are defined and documented as lower case.
